src/components/progress-circle/progress-circle.ts

```diff
@@ -41,12 +42,22 @@ type EasingFn = (currentTime: number, startValue: number,
   styleUrls: ['progress-circle.css'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class MdProgressCircle {
+export class MdProgressCircle implements OnDestroy {
   /** The id of the last requested animation. */
   private _lastAnimationId: number = 0;
 
-  /** The id of the indeterminate interval. */
+  /**
+   * The id of the indeterminate interval.
+   * @internal
```

Comment:
The `@internal` actually goes on the public member rather than the private one; this tells TypeScript to omit the property when outputting a `.d.ts` file so that downstream users can't depend on it.